meta-luneui/recipes-webos/nyx-lib/nyx-lib.bb

```diff
@@ -21,17 +21,17 @@ PV = "7.3.0-1+git${SRCPV}"
 
 DEPENDS = "glib-2.0 pmloglib"
 
-inherit webos_public_repo
+inherit webos_ports_repo
 inherit webos_cmake
 inherit systemd
 
 SYSTEMD_PACKAGES = "${PN}"
 SYSTEMD_SERVICE_${PN} = "nyx.target"
 
-SRC_URI = "git://github.com/herrie82/nyx-lib.git;branch=herrie/webosose"
+SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE};branch=webOS-ports/webOS-OSE"
```

Comment:
instead of setting ;branch=something after ${WEBOS_PORTS_GIT_REPO_COMPLETE} you should just set WEBOS_GIT_PARAM_BRANCH = "something"